Makefile.am

```diff
@@ -87,6 +87,7 @@ csharp_EXTRA_DIST=                                                           \
   csharp/src/AddressBook/SampleUsage.cs                                      \
   csharp/src/Google.Protobuf.Benchmarks/BenchmarkMessage1Proto3.cs           \
   csharp/src/Google.Protobuf.Benchmarks/Benchmarks.cs                        \
+  csharp/src/Google.Protobuf.Benchmarks/GoogleMssageBenchmarkcs              \
```

Comment:
`make[1]: *** No rule to make target 'csharp/src/Google.Protobuf.Benchmarks/GoogleMssageBenchmarkcs', needed by 'distdir'.  Stop.`